diwako_cbrn.vr/scripts/cbrn/functions/fn_fogPFH.sqf

```diff
@@ -0,0 +1,60 @@
+if (isGamePaused) exitWith {};
+params ["_args", "_idPFH"];
+_args params ["_lastIteration"];
+
+private _player = ace_player;
+
+private _time = cba_missiontime;
+private _delta = _time - _lastIteration;
+_args set [0, _time];
+
+private _uptime = _player getVariable ["cbrn_mask_uptime", 0];
+private _fogging = _player getVariable ["cbrn_mask_fogging", false];
+
+private _fatigue = _player call cbrn_fnc_getFatigue;
+private _fatigueAdditive = 0;
+
+if (cbrn_fogFatigueEnabled) then {
+	_fatigueAdditive = ( _fatigue * cbrn_fogFatigueCoef ) * _delta;
+};
+
+if (_player getVariable["cbrn_mask_on", false]) then {
```

Comment:
Push this up way higher, no need to grab anything like up time, fogging or fatigue if you do nothave a mask on

Reply:
also tabs -> spaces